users/views.py

```diff
@@ -1,3 +1,69 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from .models     import User
+from .utils import email_validation, password_validation
+from my_settings import SECRET_KEY, ALGORITHM    
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            full_name       = data["full_name"]
+            email           = data["email"]
+            membership      = data["membership"]
+            address         = data["address"]
+            phone_number    = data["phone_number"]
+            gender_id       = data["gender_id"]
+            password        = data["password"]
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+            if not email_validation(email):
+                return JsonResponse({"MESSAGE" : "INVALID_EMAIL"})
+            elif not password_validation(password):
+                return JsonResponse({"MESSAGE" : "INVALID_PASSWORD"})
```

Comment:
- 반환해줄 http status code 추가해주세요.
- 여기서는 굳이 elif가 필요하지 않고 전부 if문으로 변경해주셔도 좋습니다. 
- 어짜피 특정조건이 truthy 하다면 무조건 return문을 만나기 때문에 elif 나 else는 불필요합니다.
```suggestion
            if not email_validation(email):
                return JsonResponse({"MESSAGE" : "INVALID_EMAIL"}, status=XXX)
                
            if not password_validation(password):
                return JsonResponse({"MESSAGE" : "INVALID_PASSWORD"}, status=XXX)
```

Reply:
수정하였습니다~